RHEL6 - e2e test BridgeNetwork fails (bridge not supported)

### Version of Singularity:

What version of Singularity are you using? Run:

```
3.5.2
```

### Actual behavior

```
--- FAIL: TestE2E/PAR/ACTIONS/network (1.76s)
                --- FAIL: TestE2E/PAR/ACTIONS/network/BridgeNetwork (0.16s)
                    actions.go:1009: Running command "/usr/local/bin/singularity --debug exec --net --network bridge /tmp/stest.490211207/test.s
if id"
                    actions.go:1009: 
                        "/usr/local/bin/singularity --debug exec --net --network bridge /tmp/stest.490211207/test.sif id" output:
                        ESC[0mDEBUG   ESC[0m[U=0,P=22498]      persistentPreRunE()           Singularity version: 3.5.2+141-g7742e82
                        ESC[0mDEBUG   ESC[0m[U=0,P=22498]      handleConfDir()               /root/.singularity already exists. Not creating.
                        ESC[0mDEBUG   ESC[0m[U=0,P=22498]      initCacheDir()                Creating cache directory: /tmp/stest.490211207/e2e-
imgcache-324163602/cache
                        ESC[0mDEBUG   ESC[0m[U=0,P=22498]      initCacheDir()                Creating cache directory: /tmp/stest.490211207/e2e-
imgcache-324163602/cache/library
                        ESC[0mDEBUG   ESC[0m[U=0,P=22498]      updateCacheSubdir()           Caching directory set to /tmp/stest.490211207/e2e-i
mgcache-324163602/cache/library
                        ESC[0mDEBUG   ESC[0m[U=0,P=22498]      initCacheDir()                Creating cache directory: /tmp/stest.490211207/e2e-
imgcache-324163602/cache/oci-tmp
...
```



<!-- if this is a feature request, you can ignore this next part -->

### Steps to reproduce this behavior

How can others reproduce this issue/problem?


### What OS/distro are you running

```
RHEL 6
```

--- a/e2e/actions/actions.go
+++ b/e2e/actions/actions.go
@@ -984,7 +984,7 @@ func (c actionTests) actionBasicProfiles(t *testing.T) {
 func (c actionTests) actionNetwork(t *testing.T) {
 	e2e.EnsureImage(t, c.env)
 
-	e2e.Privileged(require.Network)
+	e2e.Privileged(require.Network)(t)
 
 	tests := []struct {
 		name       string

